cpp/unreal_plugins/SimulationController/Source/SimulationController/SonarSensor.cpp

```diff
@@ -1,56 +1,58 @@
-////------ BEGIN UE5 MIGRATION ------////
-//// Uncomment this file when OpenBot is supported in UE5.
+// TODO: uncomment when we're ready to re-enable Sonar sensor
```

Comment:
Let's remove `SonarSensor` because we're approaching the 0.3.0 release. We can always recover it later when we're ready to clean it up and/or generalize it into a general-purpose ray-casting sensor.